docker/scripts/mininet/multi_switch_mininet.py

```diff
@@ -79,9 +81,11 @@ def main():
     conf = manifest['targets'][args.target]
     params = conf['parameters'] if 'parameters' in conf else {}
 
+    os.environ.update(dict(map(lambda (k,v): (k, str(v)), params.iteritems())))
+
     def formatParams(s):
         for param in params:
-            s = s.replace('%'+param+'%', str(params[param]))
+            s = s.replace('{'+param+'}', str(params[param]))
```

Comment:
Do we still need to do our own formatting, or is the shell enough and we could just use `$foo` here? It looks like you're using this format when specifying links in the manifest file, so I'm guessing that stuff doesn't get processed by the shell? If not, it'd be nice to at least use the same syntax. (`$foo` is ideal, but `${foo}` also works in the shell so that's a possible alternative.)